invenio_vocabularies/config.py

```diff
@@ -175,6 +185,9 @@ def is_pic(val):
 SUBJECTS_EUROSCIVOC_FILE_URL = "https://publications.europa.eu/resource/distribution/euroscivoc/rdf/skos_ap_eu/EuroSciVoc-skos-ap-eu.rdf"
 """Subject EuroSciVoc file download link."""
 
+AFFILIATIONS_EDMO_SPARQL_URL = "https://edmo.seadatanet.org/sparql/sparql"
```

Comment:
Minor: this config does not start with `VOCABULARIES_` and therefore it cannot be overridden (similarly to other variables above). It is probably wanted... but then in this case, do we want to have those in the `config.py`?
It feels like that they are more `const` and not really config vars.